ai/builders/gateways.mdx

```diff
@@ -0,0 +1,723 @@
+---
+title: AI Gateways
+---
+
+There are multiple AI gateways available in the Livepeer ecosystem that you can
+use to interact with the Livepeer AI network. Each gateway has its own unique
+features and capabilities. Here are some of the gateways that you can use to
+interact with the Livepeer AI network.
+
+<CardGroup cols={2}>
+  <Card
+    title="Livepeer Cloud Community Gateway"
+    href="https://www.livepeer.cloud/"
+    icon={
+      <svg
```

Comment:
@victorges sadly mintlify doesn't yet accept svg paths. I submitted a feature request with the Mintlify team (see https://mintlify-users.slack.com/archives/C05D00NS55F/p1726563283058219?thread_ts=1726305209.436119&cid=C05D00NS55F).